bin/plugin/open/selfAddIngressKey

```diff
@@ -67,14 +67,20 @@ if (not defined $pubKey) {
     my $algos    = join(' ', @algoList);
     osh_info "Please paste the SSH key you want to add. This bastion supports the following algorithms:\n";
 
+    if (grep { 'ed25519-sk' eq $_ } @algoList) {
+        osh_info "ED25519 Secure-Key: strongness[######] speed[#####], use `ssh-keygen -t ed25519-sk' to generate one";
+    }
     if (grep { 'ed25519' eq $_ } @algoList) {
-        osh_info "ED25519: strongness[#####] speed[#####], use `ssh-keygen -t ed25519' to generate one";
+        osh_info "ED25519: strongness[#####.] speed[#####], use `ssh-keygen -t ed25519' to generate one";
+    }
+    if (grep { 'ecdsa-sk' eq $_ } @algoList) {
+        osh_info "NIST-P Secure key: strongness[#####.] speed[#####], use `ssh-keygen -t ecdsa-sk -b 521' to generate one";
```

Comment:
I think we should be consistent with "ECDSA" here (which, indeed, is also a nist-p curve). I'm afraid it would be confusing to users otherwise

Reply:
Ack